nativelib/src/main/scala-next/scala/scalanative/safe/SafeZone.scala

```diff
@@ -0,0 +1,51 @@
+package scala.scalanative.safe
```

Comment:
I'm not sure if `safe` package would be the best name. Maybe we can change it to `scala.scalanative.memory` or even move it  to `scala.scalanative` to make it easily accessible.